pkg/controller/plan/validation.go

```diff
@@ -317,8 +318,8 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 		Type:     NameNotValid,
 		Status:   True,
 		Reason:   NotValid,
-		Category: Critical,
-		Message:  "Target VM name not valid.",
+		Category: Warn,
+		Message:  "Target VM name not valid, will be automatically changed.",
```

Comment:
maybe here we can say based on DNS1123 RFC convention

Reply:
or: Target VM name does not comply with DNS1123 RFC, ...